### ISSUE TITLE
Can we disable saving the qrcode img to file?

### ISSUE THREAD

**Issue by huan** [wechaty/wechaty-puppet-oicq#7]
```
请用手机QQ扫描二维码，若打印出错请打开：/home/huan/git/wechaty/wechaty-getting-started/data/12345/qrcode.png
```

The above image file is not necessary to be created and will generate rubbish files.

Can we disable this feature for the puppet?

**Comment by anaivebird** [comment]
qrcode.png with device-{QQnumber}.json will be created on data/{QQnumber} folder.

![image](https://user-images.githubusercontent.com/29398611/135813353-e85f33fe-f0db-40c0-b582-101bb29c6f2c.png)

The device json will save random device info for a enumlated phone device.
If we keep the device.json, and login for several days.  Our QQ accound will be less likely to  be banned since QQ server records we login constantly with same IP and device.

Also, since the exist of "device lock(设备锁)", we need to keep the device.json to prevent of allowing new device to login in each time.

![image](https://user-images.githubusercontent.com/29398611/135813747-3e603d0f-75d5-4bea-a998-a265ee0bc683.png)

**Comment by huan** [comment]
Yes, we should save the device.json and keep it for reuse across sessions.

That information should be saved to MemoryCard that managed by Wechaty.

Please read the below lines of source code to understand how to use MemoryCard (`puppet.memory.set()`) :

https://github.com/wechaty/wechaty-puppet-wechat/blob/a45a4ab7e20bb9cfc328e904966a550dbdc8e3a0/src/puppet-wechat.ts#L1219

To summarize: the `puppet.memory` is managed by the Wechaty Puppet system, you can save/load critical data from and to it across sessions.
